roles/pulp_webserver/tasks/nginx.yml

```diff
@@ -62,6 +62,8 @@
         loop_var: __pulp_webserver_snippet
       when: __pulp_webserver_snippet.rc == 0
       notify: reload nginx
+      tags:
+        - molecule-idempotence-notest
```

Comment:
This is part of being idempotent. Why wouldn't we test it?

Reply:
Galaxy = 1 plugin on 1st run,
3 plugins on 2nd run